2021-07-09 03:03:11,119 ERROR 1660 nodejs.ClusterClientNoResponseError: client no response in 3874154ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\Users\七环\Desktop\mall\server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 1660
hostname: LAPTOP-F0FF6E0L

2021-07-09 04:51:55,633 ERROR 1660 nodejs.ClusterClientNoResponseError: client no response in 6484488ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\Users\七环\Desktop\mall\server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 1660
hostname: LAPTOP-F0FF6E0L

2021-07-09 06:22:06,776 ERROR 1660 nodejs.ClusterClientNoResponseError: client no response in 2850176ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\Users\七环\Desktop\mall\server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 1660
hostname: LAPTOP-F0FF6E0L

2021-07-09 21:45:03,563 ERROR 3632 nodejs.ClusterClientNoResponseError: client no response in 9928153ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\Users\七环\Desktop\mall\server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 3632
hostname: LAPTOP-F0FF6E0L

2021-07-09 22:02:24,197 ERROR 3632 nodejs.ClusterClientNoResponseError: client no response in 688437ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\Users\七环\Desktop\mall\server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 3632
hostname: LAPTOP-F0FF6E0L

2021-07-09 22:35:04,564 ERROR 3632 nodejs.ClusterClientNoResponseError: client no response in 1940338ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\Users\七环\Desktop\mall\server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 3632
hostname: LAPTOP-F0FF6E0L

2021-07-09 23:02:41,459 ERROR 3632 nodejs.ClusterClientNoResponseError: client no response in 1036593ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (E:\Users\七环\Desktop\mall\server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 3632
hostname: LAPTOP-F0FF6E0L

